Pinta.Effects/Effects/EmbossEffect.cs

```diff
@@ -100,31 +100,30 @@ public override void Render (ImageSurface src, ImageSurface dst, Core.RectangleI
 	}
 
 
-	public double[,] Weights {
-		get {
-			// adjust and convert angle to radians
-			double r = (double) Data.Angle * 2.0 * Math.PI / 360.0;
+	public double[,] Weights ()
```

Comment:
Maybe name this something like `ComputeWeights()` or `BuildWeights()`

Reply:
No problem, I just did it